src/main/webapp/app/exam/monitoring/charts/activity-log/total-actions-chart.component.ts

```diff
@@ -13,6 +15,8 @@ import { ActivatedRoute } from '@angular/router';
 export class TotalActionsChartComponent extends ChartComponent implements OnInit, OnDestroy {
     readonly renderRate = 10;
 
+    actionsPerTimestamp: Map<string, number> = new Map();
```

Comment:
Could you also use actual dates as keys to avoid the conversion later on?
This should also simplify potential changes later on that need the actual dates

Reply:
I need to convert it at some point, since grouping by returns a string as the key. So I decided to save it as a string and convert it later. This way I avoid the double conversion since I need it as a string and date.